db/patterns/naver.com.eno

```diff
@@ -6,11 +6,15 @@ organization: naver
 --- domains
 naver.com
 naver.net
+pstatic.net
```

Comment:
This here breaks the tests (domains not being unique). It turns out we have an entry (Naver CDN) already for this domain:

https://github.com/ghostery/trackerdb/blob/main/db/patterns/pstatic.net.eno

Reply:
Thanks for letting me know!